docs/introduction.md

```diff
@@ -62,6 +62,20 @@ You can share class data between running VMs, which can reduce the startup time
 ### Native data operations
 If your Java application manipulates native data, consider writing your application to take advantage of methods in the Data Access Accelerator API.
 
+The following functions are provided:
+
+- Arithmetic, copying, and shifting operations for packed decimal data.
+- Routines to convert between decimal data types stored in byte arrays and Java binary types.
+- Marshalling operations: marshalling and unmarshalling Java binary types, such as `short`, `int`, `long`, `float`, and `double`, to and from byte arrays.
+
+You can gain a number of benefits by using the APIs provided:
+
+- Improved application performance by avoiding object creation and intermediate processing, which can also put pressure on the Java heap.
+- Hardware acceleration by automatically exploiting any known and available hardware features on specific platforms.
+- Platform independence for applications that are developed to take advantage of DAA.
```

Comment:
Maybe we should expand DAA acronym here or add `(DAA)` above in the first sentence of this section.